utils/chunked_vector.hh

```diff
@@ -397,6 +423,10 @@ chunked_vector<T, max_contiguous_allocation>::new_chunk(size_t n) {
 template <typename T, size_t max_contiguous_allocation>
 void
 chunked_vector<T, max_contiguous_allocation>::migrate(T* begin, T* end, T* result) {
+    if constexpr (std::is_trivially_move_constructible_v<T> && std::is_trivially_destructible_v<T>) {
+        std::memcpy(result, begin, reinterpret_cast<std::uintptr_t>(end) - reinterpret_cast<std::uintptr_t>(begin));
+        return;
+    }
```

Comment:
Here, we can use std::uninitialized_move() followed by std::destroy().

Reply:
done in v2